scalameta/src/main/scala/scala/meta/internal/parsers/Parsers.scala

```diff
@@ -623,12 +635,6 @@ private[meta] class Parser(val input: Input)(implicit val dialect: Dialect) { pa
   @inline final def commaSeparated[T <: Tree : AstMetadata](part: => T): List[T] =
     tokenSeparated[`,`, T](sepFirst = false, part)
```

Comment:
When I used `constructR1Quasi`, both methods `caseSeparated` and `readAnnots` got 0 usages (as Intellij says), so I removed them. But `commaSeparated` had at least 10 usages, and was no real need to rewrite them all.